.sample.env

```diff
@@ -1,3 +1,2 @@
-# http://ddollar.github.com/foreman/
```

Comment:
Silly thing, but I think let's bring this line back, just because it's not related to anything else, and it will  reduce the diff noise.

Reply:
Makes sense, it'll restore it! We can update it in a separate PR. That link is currently responding with "Not Found"